app/assets/stylesheets/application.scss

```
@@ -172,3 +172,16 @@ details {
   font-size: 8pt;
   color: green;
 }
+
+.search-bar {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.search-form > form {
+  margin-bottom: -20px;
+}
+
+.search-form > form > input[type=text]{
+  width: 300px;
+}
```

Nest these? I think the syntax is:
```suggestion
.search-form > form {
  margin-bottom: -20px;
  & > input[type=text] {
    width: 300px;
  }
}
```

---

Good catch. Done.